puppet-foreman_scap_client/puppet-foreman_scap_client.spec

```diff
@@ -0,0 +1,44 @@
+%global puppet_module foreman_scap_client
+%global puppet_full_name theforeman-%{puppet_module}
+
+%global puppet_modules_dir %{_datadir}/puppet/modules
+
+Name:       puppet-%{puppet_module}
+Version:    0.3.11
+Release:    1%{?dist}
+Summary:    Puppet module to configure foreman_scap_client
+License:    GPLv2
+URL:        https://github.com/theforeman/%{name}
+Source0:    https://forgeapi.puppetlabs.com/v3/files/%{puppet_full_name}-%{version}.tar.gz
+BuildArch:  noarch
+Requires:   puppet >= 2.7.0
+Requires:   puppetlabs-stdlib >= 4.2.0
```

Comment:
seems it got into epel 7

https://dl.fedoraproject.org/pub/epel/7/x86_64/p/puppetlabs-stdlib-4.5.1-2.20150121git7a91f20.el7.noarch.rpm